core/src/test/scala/za/co/absa/spline/harvester/conf/StandardSplineConfigurationStackSpec.scala

```diff
@@ -16,13 +16,25 @@
 
 package za.co.absa.spline.harvester.conf
 
+import org.apache.spark.sql.SparkSession
 import org.apache.spark.{SparkConf, SparkContext}
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
-import za.co.absa.commons.spark.SparkTestBase
 import za.co.absa.spline.harvester.conf.StandardSplineConfigurationStackSpec._
 
-class StandardSplineConfigurationStackSpec extends AnyFlatSpec with Matchers with SparkTestBase {
+class StandardSplineConfigurationStackSpec extends AnyFlatSpec with Matchers {
+
+  implicit val spark: SparkSession = SparkSession.builder()
+    .master("local[*]")
+    .appName(s"Commons unit testing SchemaUtils")
+    .config("spark.ui.enabled", "false")
+    .config("spark.debug.maxToStringFields", 100)
+    .config("spark.driver.bindAddress", "127.0.0.1")
+    .config("spark.driver.host", "127.0.0.1")
+    .config("spark.sql.hive.convertMetastoreParquet", false)
+    .config("fs.defaultFS","file:/")
```

Comment:
why are all those properties there? Can we remove those that aren't relevant for the test suite?

Reply:
Those are copied from `SparkTestBase`. Since this test class is removed in `develop` branch I haven't spent much time on it.

I guess `spark.debug.maxToStringFields`, `spark.sql.hive.convertMetastoreParquet` and `fs.defaultFS` could be removed.